proposed/http-message.md

```diff
@@ -1100,7 +1106,8 @@ interface StreamableInterface
      *
      * @see seek()
      * @link http://www.php.net/manual/en/function.fseek.php
-     * @return bool Returns TRUE on success or FALSE on failure.
+     * @return bool Returns TRUE on success.
```

Comment:
@mtdowling `bool` is specified because the `true` and `false` values are not valid as annotation types. So, `@return bool Returns TRUE` is how we signal `true` as a return value.